cleverhans/attacks.py

```diff
@@ -210,6 +211,9 @@ def generate(self, x, **kwargs):
                   labels to avoid the "label leaking" effect (explained in this
                   paper: https://arxiv.org/abs/1611.01236). Default is None.
                   Labels should be one-hot-encoded.
+        :param y_target: (optional) A tensor with the labels to target. Do not
+                         set y_target if y is also set. Labels should be
```

Comment:
I've slightly improved it to the first version. If you want to try to improve it more feel free to commit to the branch.